React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -132,6 +136,15 @@ - (void)presentViewController:(UIViewController *)alertController
     index++;
   }
 
+  for (NSNumber *disabledButtonIndex in disabledButtonIndices) {
+    if ([disabledButtonIndex integerValue] < buttons.count) {
+      [alertController.actions[[disabledButtonIndex integerValue]] setEnabled:false];
+    } else {
+        RCTLogError(@"Index %li is out of bounds. Maximum index value is %lu.", (long)[disabledButtonIndex integerValue], buttons.count - 1);
```

Comment:
it'd be nice if the error contained some information about what it's related to (disabledButtonIndices)

Reply:
Let's say: `Index X from disabledButtonIndices is out of bounds. The maximum index value is 4.` Wdyt?

Reply:
yes, that sounds good to me 👍 
but please note I'm not a person who can merge the PR :)